### ISSUE TITLE
Product gruop 

### ISSUE THREAD

**Issue by Gipielle** [PrestaShop/PrestaShop#18337]
Hello,

I would like to be able to group the products but not as combinations, but as a group of articles that have a common description of images but with different codes and prices.
The problem with the combinations lies in the fact that I have to calculate all the prices according to the base price and I cannot set the price of each single product.

We allow the sale of screws of different sizes
in prestashop I have 2 solutions:
create 50 practically identical products unlike the size
or create a combination, but in this case I have to calculate all the prices and I can't set the price I want.

Basically the function for the combination would be fine with the exception of price and feature management.

Having products whose characteristics are the same but for example change the weight, or the length, or any other size, using the characteristics for the combinations also generates many combinations that in fact have no use.

Is there any solution ?
Or maybe I'm just wrong and prestashop already manages this.

Thanks

**Comment by sarahdib** [comment]
Hello, 

Please be aware that this bug tracker should be used to report issues only. 
If you need help, you can ask on the :
- Forum (https://www.prestashop.com/forums/)
- Gitter (https://gitter.im/PrestaShop/General)  
- Stack Overflow (https://stackoverflow.com/questions/tagged/prestashop)

Thanks for your understanding!

**Comment by Gipielle** [comment]
Sorry a choose issue instead of feature request
